packages/generator/src/GenerateContext/EnhancedBase.ts

```diff
@@ -55,9 +55,7 @@ export abstract class AbstractImportable {
     this.importPath = this.isLocal
       ? `${basePath}/${this.shortApiName}${importExt}`
       : this.sourcePackage!;
-    this.uniqueImportName = this.isLocal
-      ? this.shortApiName
-      : `$external$${this.fullApiName.replace(".", "$")}`;
+    this.uniqueImportName = this.shortApiName;
```

Comment:
Why no need for the local check anymore?

Reply:
The functions I put on the object handle this now.